src/io/flutter/jxbrowser/JxBrowserManager.java

```diff
@@ -133,11 +133,11 @@ public void setUp(Project project) {
     // Retrieve key
     try {
       final String key = JxBrowserUtils.getJxBrowserKey();
-      System.setProperty(JxBrowserUtils.PROPERTY_NAME, key);
+      System.setProperty(JxBrowserUtils.LICENSE_PROPERTY_NAME, key);
     }
     catch (FileNotFoundException e) {
       LOG.info(e.getMessage());
-      LOG.info(project.getName() + ": Unable to find JxBrowser licence key file");
+      LOG.info(project.getName() + ": Unable to find JxBrowser license key file");
```

Comment:
You may be able to use `LOG.info(project.getName() + ..., e)` instead of two separate log calls.